sabnzbd/notifier.py

```diff
@@ -265,6 +285,92 @@ def send_prowl(title, msg, notification_type, force=False, test=None):
     return ""
 
 
+def send_apprise(title, msg, notification_type, force=False, test=None):
+    """send apprise message"""
+    logging.debug("Sending Apprise notification")
+    if test:
+        urls = test.get("apprise_urls")
+    else:
+        urls = sabnzbd.cfg.apprise_urls()
+
+    # Notification mapper
+    n_map = {
+        # Startup/Shutdown
+        "startup": apprise.common.NotifyType.INFO,
+        # Pause/Resume
+        "pause_resume": apprise.common.NotifyType.INFO,
+        # Added NZB
+        "download": apprise.common.NotifyType.INFO,
+        # Post-processing started
+        "pp": apprise.common.NotifyType.INFO,
+        # Job finished
+        "complete": apprise.common.NotifyType.SUCCESS,
+        # Job failed
+        "failed": apprise.common.NotifyType.FAILURE,
+        # Warning
+        "warning": apprise.common.NotifyType.WARNING,
+        # Error
+        "error": apprise.common.NotifyType.FAILURE,
+        # Disk full
+        "disk_full": apprise.common.NotifyType.WARNING,
+        # Queue finished
+        "queue_done": apprise.common.NotifyType.INFO,
+        # User logged in
+        "new_login": apprise.common.NotifyType.INFO,
+        # Other Messages
+        "other": apprise.common.NotifyType.INFO,
+    }
+
+    # Prepare our Asset Object
+    asset = apprise.AppriseAsset(
+        app_id="SABnzbd",
+        app_desc="SABnzbd Notification",
+        app_url="https://sabnzbd.org/",
+        image_path_mask=os.path.join(os.path.dirname(__file__), "apprise", "apprise-{TYPE}.png"),
```

Comment:
I don't think we should have any images in the `/sabnzbd` directory, it's only for code.
We should put it in `/icons/apprise`.

Reply:
Do you want me to hardcode `/icons/apprise` or find it relative to the directory.  This is the edge case with Apprise, there are like 70 services that just use the icons found on the web, but then there are like 30 that use the local icons stored on the server (True self-hosted vs pulling from the cloud).

This was why i originally stored the images here and then cross-referenced them here via both web and locally (it's now i do it with apprise, so it's just 1 directory to manage - [link](https://github.com/caronc/apprise/tree/master/apprise/assets/themes/default).

At the end of the day, to truly get the best use of Apprise, you need some images stored locally.  If you want, i can just not set a mask over-ride at all, and you'll use the ones that ship with Apprise instead of some custom SABnzbd ones i rigged up quickly?

Reply:
I can fix this one up when all the code is fine, we have some constants to handle this :)